tests/test_chi_ohare_noise.py

```diff
@@ -0,0 +1,451 @@
+    )
+    parsed_sub_items.append(
+        spider.ChiOhareNoiseSubSpider1()._parse_details(test_response_2)
+    )
+
+
+def test_title():
+    assert parsed_sub_items[0]["title"] == (
+        "O'Hare Noise Compatibility Commission" + " Meeting (via video/teleconference)"
+    )
+    assert parsed_sub_items[1]["title"] == (
+        "Fly Quiet Committee" + " (via video/teleconference)"
+    )
+    assert parsed_sub_items[2]["title"] == "Executive Committee Meeting"
+    assert parsed_sub_items[3]["title"] == "Executive Committee Meeting"
+    assert parsed_sub_items[4]["title"] == "Fly Quiet Committee"
+
+
+def test_description():
+    assert parsed_sub_items[0]["description"] == "Join Zoom Meeting"
+    assert parsed_sub_items[1]["description"] == "Join Zoom Meeting"
+    assert parsed_sub_items[2]["description"] == ""
+    assert parsed_sub_items[3]["description"] == ""
+    assert parsed_sub_items[4]["description"] == ""
+
+
+def test_start():
+    assert parsed_sub_items[0]["start"] == datetime(2020, 6, 5, 8, 0)
+    assert parsed_sub_items[1]["start"] == datetime(2020, 5, 26, 9, 30)
+    assert parsed_sub_items[2]["start"] == datetime(2020, 2, 10, 10, 30)
+    assert parsed_sub_items[3]["start"] == datetime(2020, 1, 6, 10, 30)
+    assert parsed_sub_items[4]["start"] == datetime(2020, 12, 8, 9, 30)
+
+
+def test_end():
+    assert parsed_sub_items[0]["end"] == datetime(2020, 6, 5, 9, 0)
+    assert parsed_sub_items[1]["end"] == datetime(2020, 5, 26, 10, 30)
+    assert parsed_sub_items[2]["end"] == datetime(2020, 2, 10, 11, 30)
+    assert parsed_sub_items[3]["end"] == datetime(2020, 1, 6, 11, 30)
+    assert parsed_sub_items[4]["end"] == datetime(2020, 12, 8, 10, 30)
+
+
+def test_status():
+    assert parsed_sub_items[0]["status"] == PASSED
+    assert parsed_sub_items[1]["status"] == PASSED
+    assert parsed_sub_items[2]["status"] == PASSED
+    assert parsed_sub_items[3]["status"] == PASSED
+    assert parsed_sub_items[4]["status"] == TENTATIVE
+
+
+def test_location():
+    assert parsed_sub_items[0]["location"] == {
+        "name": "(via video/teleconference)",
+        "address": "(via video/teleconference)",
+    }
+    assert parsed_sub_items[1]["location"] == {
+        "name": "(via video/teleconference)",
+        "address": "(via video/teleconference)",
+    }
+    assert parsed_sub_items[2]["location"] == {
+        "name": "Aviation Administration Building",
+        "address": "10510 W. Zemke Rd",
+    }
+    assert parsed_sub_items[3]["location"] == {
+        "name": "Aviation Administration Building",
+        "address": "10510 W. Zemke Rd",
+    }
+    assert parsed_sub_items[4]["location"] == {
+        "name": "Chicago Dept. of Aviation Building",
+        "address": "10510 W. Zemke Road, Chicago, IL, Conference Room 1",
+    }
+
+
+def test_source():
+    src = "https://www.oharenoise.org/about-oncc"
+    src += "/oncc-meetings/month.calendar/2019/02/03/-"
+    assert parsed_sub_items[0]["source"] == src
```

Comment:
This one is pretty safe, and we can shorten it a bit and get at what we're testing with `== test_response.url`